src/js/utils/permissionUtils.js

```diff
@@ -11,12 +11,17 @@ const canEditRequest = (currentUser, request, location) => {
     ActivityCode.APPROVE_REQUEST,
   );
 
-  // If the location supports request approval, only the requestor is able to edit
-  // and the request can't be approved / rejected when editing
   if (isApprovalRequired) {
+    // If the request is approved, approver cannot edit request,
+    // but normal warehouse user can edit it
```

Comment:
what do we mean by "normal warehouse user can edit it"?
Below I can see that you check if `isUserRequestor`, so that would mean that if request is approved, only requestor can edit that, is that true?

Reply:
I forgot that I did all of the requests from one account - so I was the only requestor here 🫣